### PR TITLE
Upgrade to 1.9.0 version of the native library

### PR DESCRIPTION
**Pull request by VladimirAmiorkov**
## PR Checklist

- [X] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [X] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [X] All existing tests are passing
- [ ] Tests for the changes are included

## What is the current behavior?

The current version uses a pre-release version 0.9.0+ of the native Android Fresco library. Also sometimes setting the `imageUri` to an incorrect value causes exception that is not descriptive enough.

## What is the new behavior?

The plugin now uses the latest official 1.9.0 version of the native Android Fresco library as documented [here](http://frescolib.org/docs/index.html).

Fixes/Implements/Closes #[Issue Number].
#59
#16 

## BREAKING CHANGES:

- `AnimatedImage` class has been removed
- `IAnimatedImage`  interface has been renamed to `AnimatedImage` 
- `IImageInfo` interface has been renamed to `ImageInfo` 
- `IError` interface has been renamed to `FrescoError` 

### Migration
- If you are using the `AnimatedImage` class you can continue to cast to this type just note that it is now an interface rather than a class
- If you are using the `IAnimatedImage` interface simply change all of your references to be `AnimatedImage`
- If you are using the `IImageInfo` interface simply change all of your references to be `ImageInfo`
- If you are using the `IError` interface simply change all of your references to be `FrescoError`

### CODE REVIEW

File: src/platforms/android/include.gradle [hunk 1]
```diff
@@ -1,11 +1,8 @@
 android {
-	productFlavors {
```
Comment (lini) [inline src/platforms/android/include.gradle]: productFlavors are needed for backwards compatibility with {N} 3.x

File: src/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-fresco",
-  "version": "3.1.0",
+  "version": "4.0.0",
```
Comment (lini) [inline src/package.json:3]: Major version is expected to have breaking changes. If it does, they should be described in a migration guide in the plugin's readme (see imagepicker for example)

**Review by lini — CHANGES_REQUESTED** [review]

File: src/platforms/android/include.gradle (hunk 1 above)
Comment (VladimirAmiorkov) [reply src/platforms/android/include.gradle]: Yes of course, this was an auto change by the tns run command

File: src/package.json (hunk 2 above)
Comment (VladimirAmiorkov) [reply src/package.json:3]: Added breaking changes to PR and readme

**Review by lini — APPROVED** [review]

**Pull request merged**